packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -315,7 +315,7 @@ async function _getDashboardStats(app) {
   function _extractVmIdsFromBackupJob(job) {
     let vmIds
     try {
-      vmIds = extractIdsFromSimplePattern(job.vms).filter(vmId => app.hasObject(vmId, 'VM'))
+      vmIds = extractIdsFromSimplePattern(job.vms)
```

Comment:
if this is not necessary anymore , I think you could add a comment to the function : " like id may not exists anymore" ( I can't make a code suggestion )